### PR TITLE
Fix eksctl download link: was downloading 0.13 instead of latest

### PR DESCRIPTION
**Pull request by fmedery**
*Issue #, if available:*

*Description of changes:*
* Updated `eksctl` url based on the latest instruction from [weaveworks](https://github.com/weaveworks/eksctl/commit/49ce9b7bf7b684ffb7f19512bf6f718ad40323ef#diff-04c6e90faac2675aa89e2176d2eec7d8)
* Minor updates and fixes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/030_eksctl/launcheks.md [hunk 1]
```diff
@@ -9,13 +9,18 @@ weight: 20
...
+
 **How do I check the IAM role on the workspace?**
 
 {{%expand "Expand here to see the solution" %}}
-Run `aws sts get-caller-identity` and validate that your _Arn_ contains `eksworkshop-admin`and an Instance Id.
+Run the command below and validate that your _Arn_ contains `eksworkshop-admin`and an Instance Id.
+
+```bash
+aws sts get-caller-identity
+```
```
Comment (brentley) [inline content/030_eksctl/launcheks.md]: This should actually probably be changed to the "output" shortcode

Comment (fmedery) [reply content/030_eksctl/launcheks.md]: Tx for the feedback, I just pushed the fix into this branch

**Comment by brentley** [comment]
Thanks @fmedery but I still don't see any questions to the answers I asked above. I'll close this for now, but we can continue the discussion and re-open if necessary.

**Comment by fmedery** [comment]
I will create new PR with updated URL only.

**Pull request closed without merging**